cmd/gemini/root.go

```diff
@@ -300,9 +314,10 @@ func run(cmd *cobra.Command, args []string) error {
 	case <-gCtx.Done():
 	default:
 		testJobs := jobsFromMode(mode)
-		for _, testJob := range testJobs {
+		for id := range testJobs {
+			j := testJobs[id]
```

Comment:
THis one is already part of the another your bugfix PR and it will cause the merge conflict.